docs/arc42/context_view.md

```diff
@@ -0,0 +1,139 @@
+# Context View
+
+## System Context
+
+The geOrchestra Gateway is a central component in the geOrchestra Spatial Data Infrastructure (SDI), acting as the primary entry point for all user interactions with the platform.
+
+### System Context Diagram
+
+The system context diagram below shows the Gateway in the context of the overall geOrchestra system and its external dependencies.
+
+![System Context Diagram](../assets/images/structurizr/structurizr-SystemContext.svg)
+
+The diagram shows the following key components and relationships:
+
+1. **Users** interact with the geOrchestra SDI platform using HTTPS
+2. **geOrchestra SDI** is the overall system containing the Gateway and other services
+3. The **geOrchestra SDI** interacts with external systems:
```

Comment:
It still feels a bit off. I'd replace "geOrchestra SDI" by Gateway, I believe it would be more meaningful. But still, I'm not sure this diagram brings much to the global understanding

Reply:
Apart from this (and, well, not really a big deal), it's all good for me. 
It's getting late here, I'll disconnect. On my side, feel free to merge it

Reply:
And thanks for your patience ! ;o)

Reply:
Nope, context view is context view. There's the [container view](https://groldan.github.io/georchestra-gateway/arc42/container_view/) where the gateway is shown right bellow that one :)